doc/release-notes-1.0.0.rst

```diff
@@ -286,7 +286,7 @@ Bluetooth Low Energy
 * The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
 * On some operating systems, the nrf_desktop application is unable to reconnect to a host.
 * central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
-* central_hids: After flashing a HEX file to the nrf52_pca10040 board, UART connectivity is lost when using the BLE Controller. The board must be reset to get UART output.
+* central_hids: After programming a HEX file to the nrf52_pca10040 board, UART connectivity is lost when using the BLE Controller. The board must be reset to get UART output.
```

Comment:
Same comment, this is also a release notes files.

Reply:
Reverted the changes.